### PR TITLE
Change Biome Lint to OxLint

### PR DESCRIPTION
**Pull request by BleedingDev**
It has better feature parity with EsLint, but is way more performant (just like Biome is).

### CODE REVIEW

File: apps/coursition/src/app/actions.ts [hunk 1]
```diff
@@ -34,19 +34,20 @@ export const createUser = async ({ email, password }: { email: string; password:
         paymentStatus: 'FREE',
       },
     })
-    .catch((err: unknown) => {
-      if (err instanceof PrismaClientKnownRequestError) {
-        if (err.code === 'P2002') {
+    .catch((error: unknown) => {
+      if (error instanceof PrismaClientKnownRequestError) {
+        if (error.code === 'P2002') {
           return { error: 'An account already exist with this email' }
```
Comment (devodii) [inline apps/coursition/src/app/actions.ts:40]: i  see now that we should always use `error` as suppoesed to `err`, is that a convention that you're trying to implement?

File: apps/coursition/src/components/buyLifetime.tsx [hunk 2]
```diff
@@ -23,14 +23,13 @@ export const BuyLifetime = ({ className, withIcon = true, ...rest }: Props) => {
...
-    const eventHandler = async () => {
+    const eventHandler = () => {
       if (isMounted) {
         window.LemonSqueezy?.Setup({
           eventHandler: (event) => {
-            console.log({ event })
             if (event?.event === 'Checkout.Success') {
               window.LemonSqueezy.Url.Close()
-              const lsLoaderElement = document.getElementsByClassName('lemonsqueezy-loader')[0]
+              const lsLoaderElement = document.querySelectorAll('lemonsqueezy-loader')[0]
```
Comment (devodii) [inline apps/coursition/src/components/buyLifetime.tsx:32]: i think we should use a period with querySelector to target the classname, or does it work this way?

File: apps/coursition/src/app/actions.ts (hunk 1 above)
Comment (BleedingDev) [reply apps/coursition/src/app/actions.ts:40]: Yes, it is recommended via OxLint. :)

File: apps/coursition/src/components/buyLifetime.tsx (hunk 2 above)
Comment (BleedingDev) [reply apps/coursition/src/components/buyLifetime.tsx:32]: Oh damn, I had it there. :D It got deleted.

**Pull request merged**